### DISCUSSION TITLE
[Support]:  Does anyone know the tag for 0.13.2, 0.14 causes no frames received on all my cameras and I'd like to downgrade

### DISCUSSION THREAD

**Discussion by aaronjrodrigues** [blakeblackshear/frigate#13042] [Ask A Question] [answered]
### Describe the problem you are having

After upgrading to 0.14 my cameras ended up not working on frigate, in the meantime till I figure out the issue I want to downgrade to 0.13.2 but I cant find the tag. Please assist me 

### Version

0.14

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
N/A
```

### FFprobe output from your camera

```shell
N/A
```

### Frigate stats

```json
N/A
```

### Operating system

Debian

### Install method

Docker Compose

### Object Detector

CPU (no coral)

### Network connection

Wired

### Camera make and model

Hikvision

**Comment by NickM-27** [comment, marked as answer]
0.13.2 is the tag
